libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/DBSchemaAccessor.java

```diff
@@ -236,6 +237,10 @@ default List<String> showTables(String schemaName) {
 
     DBTableOptions getTableOptions(String schemaName, String tableName, String ddl);
 
+    List<DBColumnGroupElement> listTableColumnGroups(String schemaName, String tableName);
+
+    List<DBColumnGroupElement> listTableColumnGroups(String schemaName, String ddl, String tableName);
```

Comment:
we'd better not provide this method~

Reply:
done, change this method into a private method.